ProteoformSuiteGUI/ProteoformSweet.cs

```diff
@@ -185,17 +207,67 @@ public void enable_neuCodeProteoformPairsToolStripMenuItem(bool setting)
 
         private void openToolStripMenuItem_Click(object sender, EventArgs e)
         {
-            MessageBox.Show("openToolStripMenuItem_Click");
+            string working_directory;
+            MessageBox.Show("Please select a folder with saved results files.");
+            DialogResult results_folder = this.resultsFolderOpen.ShowDialog();
+            if (results_folder == DialogResult.OK)
+                working_directory = this.resultsFolderOpen.SelectedPath;
+            else if (results_folder == DialogResult.Cancel) { return; }
+            else
+                working_directory = Path.GetDirectoryName(Lollipop.deconResultsFileNames[0]);
+
+            DialogResult response = MessageBox.Show("Are these results neucode labeled?", "Neucode Labeled", MessageBoxButtons.YesNo);
```

Comment:
We should save a method file with the save-all, so that we can read this from that method file.